consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeCoordinator.java

```diff
@@ -302,7 +303,9 @@ public PayloadIdentifier preparePayload(
     if (result.isSuccessful()) {
       mergeContext.putPayloadById(
           new PayloadWrapper(
-              payloadIdentifier, new BlockWithReceipts(emptyBlock, result.getReceipts())));
+              payloadIdentifier,
+              new BlockWithReceipts(emptyBlock, result.getReceipts()),
+              result.getYield().flatMap(BlockProcessingOutputs::getRequests)));
```

Comment:
can we have a `getRequests` in `BlockProcessingResult` similar to `getReceipts`?